requirements.in

```diff
@@ -4,7 +4,7 @@ ariadne_django
 celery>=5.3.6
 cerberus
 ddtrace
-Django
+Django>=4.2.14
```

Comment:
See https://github.com/codecov/codecov-api/pull/697/files

Reply:
That update was merged. Closing my PR. Django vulnerability issue in `api` repo is now resolved.